lineage/figures/figureCommon.py

```diff
@@ -159,6 +162,19 @@ def overlayCartoon(figFile, cartoonFile, x, y, scalee=1, scale_x=1, scale_y=1):
     template.append(cartoon)
     template.save(figFile)
 
+def plotLineage(lineage):
+    """
+    Makes lineage tree.
+    """
+
+    a = [Clade(lineage.full_lineage[0].obs[2]+lineage.full_lineage[0].obs[3])]
+
+    # input the root cells in the lineage
+    c = CensoredRecursive(lineage.full_lineage[0], a)
+
+    Phylo.draw(c)
+    pylab.axis('off')
+    pylab.savefig('lineage/figures/cartoons/lineageTree.svg', format='svg', bbox_inches='tight', dpi=300)
```

Comment:
I'd actually recommend putting this in the `Plotting.py` file that you will move all the plotting functions to. And then make the path to save the figure, as an argument to the function. That is, 

```python
 def plotLineage(lineage): 
```

becomes

```python
 def plotLineage(lineage, path2save): 
```

and the last line becomes:

```python
     pylab.savefig(path2save, format='svg', bbox_inches='tight', dpi=300) 
```